docs/esa_hubble/esa_hubble.rst

```diff
@@ -0,0 +1,109 @@
+
+This package allows the access to the European Space Agency Hubble Archive (EHST; http://archives.esac.esa.int/ehst/). All the HST observations available in the EHST are synchronised with the MAST services for HST reprocessed public data and corresponding metadata. Therefore, excluding proprietary data, all HST data in the EHST are identical to those in MAST.
+
+========
+Examples
+========
+
+---------------------------
+1. Getting Hubble products
+---------------------------
+
+.. code-block:: python
+
+  >>> from astroquery.esa_hubble import ESAHubble
+  >>>
+  >>> ESAHubble.get_product("J6FL25S4Q", "RAW", "raw_data_for_J6FL25S4Q.tar")
+  http://archives.esac.esa.int/ehst-sl-server/servlet/data-action?OBSERVATION_ID=J6FL25S4Q&CALIBRATION_LEVEL=RAW
+
+This will download all files for the raw calibration level of the observation 'J6FL25S4Q' and it will store them in a tar called 'raw_data_for_J6FL25S4Q.tar'.
+
+---------------------------
+2. Getting Hubble postcards
+---------------------------
+
+.. code-block:: python
+
+  >>> from astroquery.esa_hubble import ESAHubble
+  >>>
+  >>> ESAHubble.get_postcard("J6FL25S4Q", "RAW", 256, "raw_postcard_for_J6FL25S4Q.jpg")
+  http://archives.esac.esa.int/ehst-sl-server/servlet/data-action?RETRIEVAL_TYPE=POSTCARD&OBSERVATION_ID=\
+  J6FL25S4Q&CALIBRATION_LEVEL=RAW&RESOLUTION=256
+
+This will download the postcard for the observation 'J8VP03010' with low resolution (256) and it will stored in a tar called 'raw_postcard_for_J6FL25S4Q.jpg'. Resolution of 1024 is also available.
+
+Calibration levels can be RAW, CALIBRATED, PRODUCT or AUXILIARY.
+
+---------------------------
+3. Getting Hubble artifacts
+---------------------------
+
+.. code-block:: python
+
+  >>> from astroquery.esa_hubble import ESAHubble
+  >>>
+  >>> ESAHubble.get_artifact("O5HKAX030_FLT.FITS")
+  http://archives.esac.esa.int/ehst-sl-server/servlet/data-action?ARTIFACT_ID=O5HKAX030_FLT.FITS
+
+This will download the artifact 'O5HKAX030_FLT.FITS'.
+
+----------------------------------------------
+4. Querying target names in the Hubble archive
+----------------------------------------------
+
+The query_target function queries the name of the target as given by the proposer of the observations.
+
+.. code-block:: python
+
+  >>> from astroquery.esa_hubble import ESAHubble
+  >>>
+  >>> ESAHubble.query_target("m31", "m31_query.xml")
+  http://archives.esac.esa.int/ehst-sl-server/servlet/metadata-action?RESOURCE_CLASS=OBSERVATION&\
+  SELECTED_FIELDS=OBSERVATION&QUERY=(TARGET.TARGET_NAME=='m31')&RETURN_TYPE=VOTABLE
+
+This will download metadata for all observations associated with target name 'm31'. The result of the query will be stored in file 'm31_query.xml'.
```

Comment:
This doesn't match the astroquery API.  `query_target` should return an `astropy.table.Table` object, which can then be written to file.